packages/react/package.json

```diff
@@ -11,17 +11,17 @@
     "start": "utsdx watch",
     "build": "utsdx build",
     "test": "utsdx test",
-    "lint": "utsdx lint",
-    "prepublishOnly": "yarn lint && yarn test && yarn build",
     "type:check": "yarn tsc"
   },
   "name": "@sahab/react",
   "module": "dist/react.esm.js",
   "devDependencies": {
-    "@testing-library/react-hooks": "^7.0.2",
     "@sahab/utils": "^0.2.0",
```

Comment:
I know you don''t touch this but move this package to dependencies.

Reply:
why dependencies ?  It is for testing which is a devDependency